DistFiles/xMatter/bloom-xmatter-common.less

```diff
@@ -20,6 +20,9 @@
          //Just center the margin box, (for now, we're ignoring the binding) 
           left: @MarginOuter;
           width:  @PageWidth - (2 * @MarginOuter);
+          IMG { /* BL-2353 Keeps XMatter thumb images from going too wide (or too small) */
+              max-width: @PageWidth - (2 * @MarginOuter);
```

Comment:
We are intentionally just centering the cover, as binding has little or no effect on the cover.